stage/release-3_3/debian/trixie/package-model.json

```diff
@@ -0,0 +1,21 @@
+{
+  "packages": {
+    "i3status-rs": {
+      "ref": "ubuntu/v0.22.0",
```

Comment:
The refs here must be tags. Let me update the guide to include stable releases.

Reply:
Can you update the guide soon @khos2ow ?  I found the commit for the 3.3 release but it's unknown to me how these refs are generated.

https://github.com/regolith-linux/voulage/commit/9b4d897145adc1aaa9c3801fbbdda5fd9b2f95a1

Reply:
Sorry for not being around too much. I'll update the guide and let you know.